test/jdk/java/util/zip/ZipFile/TestTooManyEntries.java

```diff
@@ -71,6 +78,10 @@ public void setup() throws Exception {
         }
     }
 
+    /**
+     * Validates that an appropriate exception is thrown when the ZipFile class
```

Comment:
Please change "appropriate" to "ZipException"

Reply:
Thank you very much. I fixed it.